meson.build

```diff
@@ -1,29 +1,43 @@
-project('writeas-gtk', ['vala', 'c'],
+# Project name, programming language and version
+project (
+    'com.github.writeas.writeas-gtk',
+    'c', 'vala',
     version: '1.0.2',
-    license: 'GPL',
-    meson_version: '>=0.46.0')
+)
 
-i18n = import('i18n')
 
-build_platform = get_option('platform')
-if build_platform == 'elementary'
-  app_id = 'com.github.writeas.writeas-gtk'
-else
-  app_id = 'writeas-gtk'
-endif
+# GNOME module
+gnome = import ('gnome')
 
-conf = configuration_data()
-conf.set_quoted('GETTEXT_PACKAGE', app_id)
-conf.set_quoted('APP_ID', app_id)
-conf.set_quoted('BUILD_PLATFORM', build_platform)
-config_h = configure_file(output: 'config.h', configuration: conf)
-config_h_dir = include_directories('.')
+# Translation module
+i18n = import ('i18n')
 
-run_target('build', command: 'meson/build-cli.sh')
```

Comment:
Does the CLI still get built when building the app now?

Reply:
I added this line back but don't see it in action. Any idea?

Reply:
Writeas CLI is not installing